### PR TITLE
CC UI improvements

### PR DESCRIPTION
**Pull request by naira-cliqz**

### CODE REVIEW

File: Cliqz/Privacy/UI/ControlCenter/OverviewViewController.swift [hunk 1]
```diff
@@ -77,18 +81,19 @@ class BlockedRequestsView: UIView {
...
-		switchControl.onTintColor = UIColor.cliqzBlueTwoSecondary
-		switchControl.thumbTintColor = UIColor.cliqzURLBarColor
+		titleLabel.textColor = UIColor.cliqzBluePrimary
+		titleLabel.font = UIFont.boldSystemFont(ofSize: 18)
+		switchControl.onTintColor = UIColor.cliqzBluePrimary
 		switchControl.transform = CGAffineTransform(scaleX: 0.75, y: 0.75)
         switchControl.addTarget(self, action: #selector(switchValueChanged), for: .valueChanged)
 		notchView.image = UIImage(named:"notch")
-		descriptionLabel.textColor = ControlCenterUI.textGray
+		descriptionLabel.textColor = UIColor(colorString: "97A4AE")
```
Comment (mahmoud-cliqz) [inline Cliqz/Privacy/UI/ControlCenter/OverviewViewController.swift]: can you move this color to `ControlCenterUI`

Comment (naira-cliqz) [reply Cliqz/Privacy/UI/ControlCenter/OverviewViewController.swift]: I don't know how to name it :)

Comment (mahmoud-cliqz) [reply Cliqz/Privacy/UI/ControlCenter/OverviewViewController.swift]: it is gray as well, right?

Comment (naira-cliqz) [reply Cliqz/Privacy/UI/ControlCenter/OverviewViewController.swift]: Yes

Comment (mahmoud-cliqz) [reply Cliqz/Privacy/UI/ControlCenter/OverviewViewController.swift]: what about `descriptionGray`

**Pull request merged**